.github/workflows/deploy-program.yml

```diff
@@ -57,6 +57,8 @@ jobs:
     steps:
       - name: Git checkout
         uses: actions/checkout@v3
+        with:
+          token: {{ secrets.SVC }}
```

Comment:
This was called `SVC_TOKEN` on other repos. Sorry for the confusion.

Reply:
Thanks! Renamed it.